website/docs/r/apigatewayv2_integration.html.markdown

```diff
@@ -79,6 +79,25 @@ resource "aws_apigatewayv2_integration" "example" {
   tls_config {
     server_name_to_verify = "example.com"
   }
+
```

Comment:
Need to add `integration_method = "ANY"` (or equivalent), else

```
aws_apigatewayv2_integration.example: Creating...

Error: error creating API Gateway v2 integration: BadRequestException: HttpMethod parameter method must be specified for integrationType HTTPPROXY
```

This error must have been present since creation but let's fix.